sunpy/net/dataretriever/sources/goes.py

```diff
@@ -191,6 +191,10 @@ def _can_handle_query(cls, *query):
                 return all(chklist)
         return False
 
+    @classmethod
+    def _attrs_module(cls):
```

Comment:
There should probably be some docs on how to write this function (ie. should be called `_attrs_module()`, should return these values...) for people wanting to do this themselves?

Reply:
I believe there is a seperate PR to update the docs.